### PR TITLE
bump dnode to 1.2.2, which plays nice with nodejs 4

### PR DESCRIPTION
**Pull request by mxk1235**
fresh off the press, dnode 1.2.2.  might as well resubmit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "node": ">=v0.8.0"
   },
   "dependencies": {
-    "dnode": "~1.2.1",
+    "dnode": "~1.2.2",
```
Comment (amir20) [inline package.json]: Wouldn't `~` already cover all minor version updates?

Comment (amir20) [inline package.json]: Ping @mxk1235.

Comment (mxk1235) [inline package.json]: oh yeah good point.  let me fix that

**Comment by amir20** [comment]
Merged. I should probably release :)

**Comment by mxk1235** [comment]
thanks!

**Pull request merged**